### DISCUSSION TITLE
提个建议

### DISCUSSION THREAD

**Discussion by zhaoqiang1024** [rime/home#1004] [Ideas]
是不是可以加一个拉取github远程配置的功能，通过github+本地双重配置来自定义用户字典，我在公司、家里，一共四台电脑每次加了新字典以后都要cp一份一另外三台上面，太麻烦了

**Comment by lotem** [comment]
用 git 就可以做到吧。

**Reply by zhaoqiang1024** [reply]
什么意思？

**Reply by lotem** [reply]
GitHub後臺使用名爲`git`的版本控制工具。用戶使用`git`客戶端即可存取託管在GitHub上的數據。不需要通過輸入法軟件。
